apps/codeforafrica/src/lib/api.ghost/posts.js

```diff
@@ -34,6 +32,26 @@ function transformPost(post) {
     month: "short",
     day: "numeric",
   });
+
+  const seo = {
+    title,
+    description: excerpt,
+    titleTemplate: `%s | ${primaryTag.slug} | CFA`,
+    openGraph: {
+      title: metaTitle || title,
+      description: metaDescription || excerpt,
+      images: [
+        {
+          url: ogImage,
+        },
+      ],
+    },
+    twitter: {
+      handle: primaryAuthor.twitter,
+      site: primaryAuthor.twitter,
```

Comment:
```suggestion
```
`site` if for who **owns** the site.

If we can pull Twitter card from ghost, its the `title` and `description` should go here.

Reply:
Yes, they're both twitter usernames but `handle` means this is the Twitter handle of the person who created the content in this page while `site` means this is the Twitter handle of the person/org/company that owns this site @kelvinkipruto